lib/engine/game/g_1824/game.rb

```diff
@@ -304,9 +345,9 @@ def init_companies(players)
           mountain_railway_count.times { |index| companies << mountain_railway_definition(index) }
 
           if option_cisleithania
-            # Remove Pre-Staatsbahn U2 and possibly U1
+            # Remove Pre-Staatsbahn U2, Coal mine C4 and possibly U1
             p2 = players.size == 2
-            companies.reject! { |m| m['sym'] == 'U2' || (p2 && m['sym'] == 'U1') }
+            companies.reject! { |m| m[:sym] == 'U2' || m[:sym] == 'SPB' || (p2 && m[:sym] == 'U1') }
```

Comment:
instead of OR'ing a bunch of corps, maybe it makes sense to put them in an array contstant, 
say REJECTED_COMPANIES, and call REJECTED_COMPANIES.include? m[:sym]